examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
```

Comment:
the `Element` function? i dunno i haven't looked closely at the source recently but it's definitely in there